README.md

```diff
@@ -20,3 +20,11 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+## Answers
+I belive that the average time complexity of my program is $\Theta$ $(n! + n)$. I found this by first looking at how many permutations are created which for a list of size $n$ which is $n!$. We also have to add the time complexity of looping through and checking if the list is sorted so thats is $n$. When we combine them we get $\Theta$ $(n! + n)$.
```

Comment:
Does the checking whether the list is sorted happen only once or for every permutation?